src/checkbox.js

```diff
@@ -0,0 +1,51 @@
+import {renderList} from './index.js';
+import _ from 'lodash'; // eslint-disable-line
+export function checkboxFunctions(){
+    let existingEntries = JSON.parse(localStorage.getItem('todos'));
+    let entry = existingEntries[this.parentNode.parentNode.id]
+
+
+    if (this.checked == true){
+        this.parentNode.parentNode.className = 'completed task';
+        entry.completed = true;
+    }else {
+        this.parentNode.parentNode.className = 'task';
+        console.log('unchecked');
+        entry.completed = false;
+    }
+    localStorage.setItem('todos', JSON.stringify(existingEntries));
+}        
+
+export function retrieveLocalStorage() {
+    if (localStorage.getItem('todos') == null){
+        console.log('No saved entries found...');
```

Comment:
- [ ] [OPTIONAL] Kindly let's remove the console.log from our code since we only need that for debugging purposes, it could be good to remove them after the debugging to keep the code less noisy 😉